### ISSUE TITLE
NVDA does not announce correctly column name in tables with merged column headers

### ISSUE THREAD

**Issue by teodord** [nvaccess/nvda#19404]
### Brief summary

In the attached PDF, the NVDA reader does not announce correctly columns "Account Type" and "Move-In Date", for the first data row of the table. It says "Column Seven", although Account Type is in fact column 5, as it correctly announces it for values in the second data row.
The problem only exist for the first data row.

[Accessibility.oo.pdf](https://github.com/user-attachments/files/24331912/Accessibility.oo.pdf)

### Steps to reproduce

Open the attached PDF with Acrobat Reader and have NVDA read through its table, which is the only piece of content in the file.

### Actual behavior

NVDA announces "Column 7 Active", for the value in the first row with data.

### Expected behavior

NVDA should announce "Account Type Column 5 Active", for the value in the first row with data.

### NVDA type

installed copy

### NVDA version

2025.3.2

### Windows version

Windows 11

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by seanbudd** [comment]
interestingly this columns an be read fine in Firefox's PDF renderer

**Comment by gerald-hartig** [comment]
@teodord Can you please provide the original source for the document so that we can rule out a PDF authoring issue.

**Comment by teodord** [comment]
Here is the ODT used to produce the PDF:

[Accessibility.oo.odt](https://github.com/user-attachments/files/24527756/Accessibility.oo.odt)
